config/initializers/assets.rb

```diff
@@ -5,10 +5,3 @@
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-# Add Yarn node_modules folder to the asset load path.
-Rails.application.config.assets.paths << Rails.root.join('node_modules')
-
```

Comment:
Reasonably sure this is a hangover from old css / asset pipeline. I can't see any assets being referenced straight from node_modules. 

Will check on preview app, let me know if there's a use case I've missed